src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo, User } from '../../types/types';
+
+type Props = {
+  todos: Todo[];
+  users: User[];
+};
+
+export const TodoInfo = ({ todos, users }: Props) => {
+  return (
+    <>
+      {todos.map(todo => {
+        const USER = users.find(user => user.id === todo.userId);
```

Comment:
```suggestion
        const user = users.find(user => user.id === todo.userId);
```
it's not a constant, so better to name it in camelCase